VisualizationBase/src/items/ViewItem.h

```diff
@@ -65,6 +67,15 @@ class VISUALIZATIONBASE_API ViewItem : public Super<DeclarativeItem<ViewItem>> {
 		QVector<QVector<Model::Node*>> nodes_;
 		QString name_;
 
+		struct LineToAdd {
+			Model::Node* from_;
+			Model::Node* to_;
+			QString layer_;
+			LineToAdd(Model::Node* from, Model::Node* to, QString layer)
```

Comment:
For a simple struct,  like the one here, this initializing constructor will be automatically generated.